ObjCVoiceCallKitQuickstart/ViewController.m

```diff
@@ -327,6 +327,8 @@ - (void)provider:(CXProvider *)provider performAnswerCallAction:(CXAnswerCallAct
     //      `provider:performAnswerCallAction:` per the WWDC examples.
     // [[TwilioVoice sharedInstance] configureAudioSession];
 
+    NSAssert([self.callInvite.uuid isEqual:action.callUUID], @"We only support one Invite at a time.");
```

Comment:
I would move this assert into `performAnswerVoiceCallWithUUID:completion:` otherwise we have don't have any reason to pass the UUID argument.